core/src/main/java/com/amplifyframework/datastore/DataStoreCategory.java

```diff
@@ -183,6 +183,16 @@ public void observe(
             onObservationStarted, onDataStoreItemChange, onObservationFailure, onObservationCompleted);
     }
 
+    @Override
+    public void start(@NonNull Action onComplete, @NonNull Consumer<DataStoreException> onError) {
+        getSelectedPlugin().start(onComplete, onError);
+    }
+
+    @Override
+    public void stop(@NonNull Action onComplete, @NonNull Consumer<DataStoreException> onError) {
+        getSelectedPlugin().stop(onComplete, onError);
+    }
+
```

Comment:
Maybe we should order these nearer to `configure(...)` and `initialize(...)` in the document, since they are more lifecycle-oriented?